vendor/plist_to_html/Makefile

```diff
@@ -0,0 +1,23 @@
+CODEMIRROR = https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.30.0
+
+BUILD_DIR = build
+BIN_DIR = $(BUILD_DIR)/bin
+DIST_DIR = $(BUILD_DIR)/dist
+
+all: package
+
+package: dep
+	mkdir -p $(BIN_DIR)
+	cp plist_to_html/dist/layout.html $(DIST_DIR)
+	cp plist_to_html/dist/css/style.css $(DIST_DIR)
+	cp plist_to_html/dist/js/bugviewer.js $(DIST_DIR)
+	cp plist_to_html/PlistToHtml.py $(BIN_DIR)/plist-to-html
+
+dep:
+	mkdir -p $(DIST_DIR)
+	curl -s -S -L -f $(CODEMIRROR)/codemirror.min.js -z $(DIST_DIR)/codemirror.min.js -o $(DIST_DIR)/codemirror.min.js
```

Comment:
Please check the curl arguments to be in sync with this:
https://github.com/Ericsson/codechecker/blob/3a68635d789276cace2ada19c67534e04c1384ff/scripts/build_package.py#L286-L291